### PR TITLE
Refactor Language Server Test cases

### PR DESCRIPTION
**Pull request by nadeeshaan**

### CODE REVIEW

File: language-server/modules/langserver-core/samples/hover/hover.bal [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@Description{value:"Test function to show current package works"}
+@Param{value:"s: string parameter"}
+@Param{value:"sd: int parameter"}
+@Return{value:"return an int"}
+function test1 (string s, int sd) returns int{
+    int a = 0;
+    return a;
+}
+
+
```
Comment (djkevincr) [inline language-server/modules/langserver-core/samples/hover/hover.bal]: May be we can get rid of this unnecessary new lines.

File: language-server/modules/langserver-core/samples/hover/hover.bal [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
```
Comment (djkevincr) [inline language-server/modules/langserver-core/samples/hover/hover.bal]: May be we can get rid of this unnecessary new lines.

Comment (nadeeshaan) [reply language-server/modules/langserver-core/samples/hover/hover.bal]: @djkevincr these new lines are specifically added to verify the line calculation logic. @NipunaMarcus might be able to confirm further

Comment (NipunaMarcus) [reply language-server/modules/langserver-core/samples/hover/hover.bal]: Let's keep them for now.. ill remove them ...

**Pull request merged**